commerce_coordinator/apps/titan/clients.py

```diff
@@ -0,0 +1,104 @@
+"""
+API clients for Titan.
+"""
+from urllib.parse import urljoin
+
+import requests
+from celery.utils.log import get_task_logger
+from django.conf import settings
+from edx_rest_api_client.client import OAuthAPIClient
+
+# Use special Celery logger for tasks client calls.
+logger = get_task_logger(__name__)
+
+
+class TitanAPIClient():
+    """
+    API client for calls to Titan using API key.
+    """
+
+    def __init__(self):
+        self.client = requests.Session()
+        # Always send API key.
+        self.client.headers.update(self.api_key_header)
+
+    @property
+    def api_base_url(self):
+        """URL of API service."""
+        return urljoin(settings.TITAN_URL, '/v1')
+
+    @property
+    def api_key_header(self):
+        """Header to add to all API requests to authenticate to endpoint."""
+        return {'X-API-Key': settings.TITAN_API_KEY}
+
+    @property
+    def normal_timeout(self):
```

Comment:
Can you move this to `core` so we can use it in other apps?

Reply:
Done: https://github.com/edx/commerce-coordinator/pull/46/commits/84d8fc879d846c93dbd05b2baa63912486ef09f2